packages/repository/src/repositories/legacy-juggler-bridge.ts

```diff
@@ -113,6 +130,10 @@ export class DefaultCrudRepository<
     );
 
     this.modelClass = this.definePersistedModel(entityClass);
+
+    // Important! Create the map object with `null` prototype to avoid
+    // Prototype Poisoning vulnerabilities.
+    this.inclusionResolvers = Object.create(null);
```

Comment:
What about using `Map`?

Reply:
Good point. I thought that objects are faster than `Map`.

I wrote a simple benchmark (http://jsben.ch/KZ25p), and found that in current Chrome `Map` is about 10% faster than objects.

Reply:
I am going to rework my code to use `Map`.

Reply:
Done in 659fee6